examples/metadata_model.py

```diff
@@ -22,54 +29,25 @@
 
 # create instance of MetadataModel class
 metadata_model_htan = MetadataModel(MM_LOC, MM_TYPE)
-print("*****************************************************")
 
-# TEST_COMP used for testing methods in 'metadata.py'
-TEST_COMP = "FollowUp"
+# TEST_COMP used for testing methods in `metadata.py`
+TEST_COMP = args.component[FIRST]
```

Comment:
change TEST_COMP to TEST_DATA_TYPE; similarly change argument name from component to data_type